src/app/pages/historia-salud/categorias/detalle-categoria.ts

```diff
@@ -67,15 +70,51 @@ export class DetalleCategoriaPage implements OnInit {
                 await alert.present();
             }
         }
-
     }
 
-    poseeAdjunto(registro) {
-        return this.categoria.descargaAdjuntos && this.getAdjunto(registro);
+    async descargarPrestacion(registro) {
+        const data = {
+            idRegistro: registro.registro.id,
+            idPrestacion: registro.idPrestacion
+        };
+        const tipo = 'pdf';
+        const pdfURL = 'modules/descargas';
+        const url1 = ENV.API_URL + `${pdfURL}/${tipo}/${data.idPrestacion}/${data.idRegistro}` + '?token=' + this.authProvider.token;
+        this.open(url1);
     }
 
     private getAdjunto(registro) {
         return registro.registro.registros.find(x => x.nombre === 'documento adjunto');
     }
 
+    open(url) {
+        if (this.platform.is('cordova')) {
+            const fileTransfer: FileTransferObject = this.transfer.create();
+            const localFile = `${File.dataDirectory}certificado.pdf`;
+            fileTransfer.download(url, localFile, true).then((entry) => {
+                new FileOpener().showOpenWithDialog(entry.toURL(), '')
```

Comment:
Pasando el mime-type hay más chance de que lo abra la app apropiada según el tipo de archivo.
```suggestion
                new FileOpener().showOpenWithDialog(entry.toURL(), 'application/pdf')
```